iceoryx_binding_c/include/iceoryx_binding_c/chunk.h

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2021 by Apex.AI Inc. All rights reserved.
```

Comment:
Please not that I considered naming it chunkheader but 1) that then the prefixes for the functions would sound weird and 2) I imagine we could add additional functionality concerning chunks here. But we could still split later into e.g. chunk.h and chunk_header.h if we deem it necessary (depends on the functionality we want to add).